src/test/test_integration_queues.py

```diff
@@ -0,0 +1,13 @@
+#This is the integration test file which i'm working on it now
+
+import sys
+sys.path.insert(1, 'D:/Logabit/pipeforce-service-template-python-1/src/service')
+sys.path.append("../service/")
```

Comment:
Please remove

Reply:
In python using sys we can access the files which are exist in another folder.
So i have used this.
But we can find other solution for that. will change this.

Reply:
https://docs.python.org/3/reference/import.html